source/includes/language-compatibility-table-java.rst

```diff
@@ -2,6 +2,7 @@
    :header-rows: 1
    :stub-columns: 1
    :class: compatibility-large
+   :widths: 20 10 10 10 10 10
```

Comment:
The first table on this page uses a different width for the version column. It looks a little odd to me that the tables use different dimensions for that column now -- can (and should) we make those widths uniform?

Reply:
Great question! I think this should just use the "compatibility-large" class and not set widths.

I think the widths will still be different even if this is removed. I checked `leafygreen-ui` and 'snooty' to try and figure out what `compatibility-large` is doing but unfortunately I couldn't find anything. 

I think this could be [hardcoded](https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#tables) to a certain width, but I think that would be different from[ other compatibility tables](https://raw.githubusercontent.com/mongodb/docs-ecosystem/master/source/includes/php-mongodb-compatibility-table.rst) I've seen in Drivers Docs. I think I'm ok leaving these with different widths, there is [somewhat of a precedent for it](https://docs.mongodb.com/drivers/cxx/#compatibility) on docs ecosystem. 

I'll ask in the DevEd channel what the `compatibility-large' class is doing.